backend/app/services/transcription.py

```
@@ -37,28 +37,66 @@ def transcribe_audio(audio_path: str, model_name: str, language: str) -> dict:
+        file_size = os.path.getsize(audio_path)
+        if file_size < 1000:  # Less than 1KB is likely invalid
+            raise RuntimeError(f"Audio file is too small ({file_size} bytes), possibly no audio in video")
+
+        logger.info(f"Audio extracted successfully: {audio_path} ({file_size} bytes)")
+        return True
+
     except subprocess.CalledProcessError as e:
-        logger.error(f"Error extracting audio: {e.stderr.decode('utf-8')}")
-        raise RuntimeError(f"Failed to extract audio: {e}")
+        error_msg = e.stderr.decode('utf-8') if e.stderr else str(e)
+        logger.error(f"Error extracting audio: {error_msg}")
```

Ruff (0.14.6)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)